test/e2e/kubectl.go

```diff
@@ -1044,6 +1045,8 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 		jobName := "e2e-test-rm-busybox-job"
 
 		It("should create a job from an image, then delete the job [Conformance]", func() {
+			// The rkt runtime doesn't support attach, see #23335
+			framework.SkipIfContainerRuntimeIs("rkt")
```

Comment:
skipping a `[Conformance]` test implies rktnetes is no longer "a kubernetes", is this a poorly written or mis-tagged test?